README.md

```diff
@@ -155,6 +121,6 @@ The target bucket will be defined by the target you've specified (`npm run deplo
 - Only `develop` branch can be deployed at the root of the `dev` bucket.
 - Only `master` branch can be deployed at the root of `int` and `prod` buckets.
 
+## Contributing
 
-### Customize VueCLI configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+See [CONTRIBUTING.md](CONTRIBUTING.md)
```

Comment:
Why not removing this and add the link to CONTRIBUTING.md directly on top of README in the Table of Content ?

Reply:
fair point, I've rearranged the order of paragraphs